bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -44,28 +44,28 @@ class GenerateAddressesOptions():
 
     Attributes:
         coinType: The type of coin.
-        range: The range of addresses to generate.
+        addr_range: The range of addresses to generate.
         bech32Hrp: The bech32 HRP (human readable part) to use.
         accountIndex: An account index.
         options: An instance of `GenerateAddressOptions`.
     """
 
     def __init__(self, coinType: CoinType,
-                 range: range,
+                 addr_range: Range,
                  bech32Hrp: str,
                  accountIndex: Optional[int] = None,
                  options: Optional[GenerateAddressOptions] = None):
         """Initialize GenerateAddressesOptions.
 
         Args:
             coinType: The type of coin.
-            range: The range of addresses to generate.
+            addr_range: The range of addresses to generate.
             bech32Hrp: The bech32 HRP (human readable part) to use.
             accountIndex: An account index.
             options: An instance of `GenerateAddressOptions`.
         """
         self.coinType = coinType
-        self.range = Range(range.start, range.stop)
+        self.addr_range = addr_range
```

Comment:
Renaming this field is wrong

Reply:
hm gonnna have to block the warning then. good catch

Reply:
well, its not giving me linter errors anymore 😂🎉